source/in-use-encryption-tutorial.txt

```diff
@@ -0,0 +1,155 @@
+.. _encryption-tutorial:
+
+==========================
+In-Use Encryption Tutorial
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In-Use Encryption allows you to connect your deployments using :v6.0:`Queryable 
+Encryption </core/queryable-encryption/>`. This connection method allows you to 
+encrypt a subset of fields in your collections. 
+
+Overview
+--------
+
+This guide shows you how to connect your deployment and collections using 
```

Comment:
[question]

Should this be "connect to your deployment"?

Reply:
Done.